docs/coordinates/example_gallery.rst

```diff
@@ -61,8 +61,8 @@ From here, we can access the components of the resulting
 `~astropy.coordinates.Galactocentric` instance to see the 3D Cartesian
 velocity components:
 
->>> print(gc1.v_x, gc1.v_y, gc1.v_z)
-30.254684717897074 km / s 171.29916086104885 km / s 18.19390627095307 km / s
+>>> print(f"{gc1.v_x:.3f}, {gc1.v_y:.3f}, {gc1.v_z:.3f}")
+30.255 km / s, 171.299 km / s, 18.194 km / s

```

Comment:
Even though this produces nicer output, isn't it going to be confusing to users in the context of the narrative? The point was to show ``gc1.v_x`` for example, but with the f string it's harder to see what is going on.

Reply:
OK reverted